src/index.ts

```diff
@@ -20,14 +20,21 @@ export default function(this: loader.LoaderContext, source: string, sourceMap?:
   options = Object.assign(defaultOptions, options);
   validateOptions(optionsSchema, options, "Coverage Istanbul Loader");
 
-  if (!sourceMap) {
-    // Check for an inline source map
-    const inlineSourceMap = convert.fromSource(source)
-      || convert.fromMapFileSource(source, path.dirname(this.resourcePath));
+  try {
+    if (!sourceMap) {
+      // Check for an inline source map
+      const inlineSourceMap = convert.fromSource(source)
+        || convert.fromMapFileSource(source, path.dirname(this.resourcePath));
 
-    if (inlineSourceMap) {
-      // Use the inline source map
-      sourceMap = inlineSourceMap.sourcemap as RawSourceMap;
+      if (inlineSourceMap) {
+        // Use the inline source map
+        sourceMap = inlineSourceMap.sourcemap as RawSourceMap;
+      }
+    }
+  } catch (e) {
+    // Exception is thrown by fromMapFileSource when there is no source map file
+    if (typeof e === "string" || e instanceof Error) {
```

Comment:
This will suppress **all** errors, not just the ones related to a missing source map.  Please check specifically for the missing-sourcemap error, and only suppress that one.  Any other error should be re-thrown.

Reply:
Solved it with `e.message.includes("An error occurred while trying to read the map file at")`

Reply:
@JamesMessinger Can you take a look at this?